hieradata/clients/private.vpn.jenkins.io.yaml

```
@@ -1,37 +1,15 @@
 ---
 profile::openvpn::networks:
   eth0:
+    # Defined in https://github.com/jenkins-infra/azure-net/blob/main/vnets.tf
     name: Subnet 'app-tier' of network 'public-vnet'
     route-metric: 100
-    # macaddress: 00:0d:3a:0e:4e:7e
-    # network_cidr: 10.0.1.0/24
     network_cidr: 10.244.0.0/14
```

```suggestion
    network_cidr: 10.248.0.0/28
```

as per https://github.com/jenkins-infra/azure-net/blob/96a2f2d4172677575624a7bcb6e96a2ca41b6cef/vnets.tf#L50